wagtailmenus/utils/misc.py

```diff
@@ -5,8 +5,12 @@
 
 
 def get_site_from_request(request, fallback_to_default=True):
-    if getattr(request, 'site', None):
+    site = getattr(request, 'site', None)
+    if site and isinstance(site, Site):
```

Comment:
Nitpick: this can be simplified to: 
```if isinstance(site, Site):```
A `NoneType` value will simply fail this test